terraform/environments/analytical-platform-ingestion/dms/dms.tf

```diff
@@ -0,0 +1,57 @@
+locals {
+    short_resource_name = "tariff-${local.environment}"
+    resource_name = "cica-ap-${local.short_resource_name}"
+}
+
+module "cica_dms_tariff_dms_implementation" {
+
+    source      = "../modules/dms"
+    vpc_id      = data.aws_vpc.connected_vpc.id
+    environment = local.environment
+
+    db          = local.short_resource_name
+
+    dms_replication_instance = {
+        replication_instance_id    = local.resource_name
+        subnet_cidrs               = local.environment_configuration.connected_vpc_private_subnets
+        subnet_group_name          = local.resource_name
+        allocated_storage          = 20
+        availability_zone          = data.aws_availability_zones.available.names[0]
+        engine_version             = "3.5.4"
+        kms_key_arn                = module.cica_dms_credentials_kms.key_arn
+        multi_az                   = false
+        replication_instance_class = "dms.t3.large"
```

Comment:
Things like `allocated_storage`, `engine_version` and `replication_instance_class` should be in environment configuration ideally